src/ClientData/include/ClientData/ScopeInfo.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_CLIENT_DATA_SCOPE_INFO_H_
+#define ORBIT_CLIENT_DATA_SCOPE_INFO_H_
+
+#include <string>
+
+namespace orbit_client_data {
+
+enum class ScopeType {
+  kInvalid = 0,
+  kDynamicallyInstrumentedFunction = 1,
+  kApiScope = 2,
+  kApiScopeAsync = 3
+};
+
+// An instance of the type uniquely identifies a scope by its name and type. The type is hashable
```

Comment:
I'm quite unsure about this sentence. It doesn't identify a specific scope, it identifies a group of scopes.

Reply:
For now -- there are no groups of scopes. When (and if) we introduce groups of scopes, I suggest to call groups of scopes scopes too.

Reply:
I didn't get it, sorry. A scope is a single slice. Multiple slices can have the same type and name, and hence are grouped in the "Live" tab.

Reply:
I'd rather use the word "scope" for the "group of slices". I don't want to avoid using the word "group" to save confusion with the group_id. Also, we already call the id for the "group of slices" `scope_id`.

Reply:
Also, what do you call "slice"? The single piece of manually instrumented code, or a single run of a manually instrumented piece of code?

Reply:
Ok, I see now. I've usually meant the single run with both slice and scope, while with scope you mean the piece of code. I'd say you are right. I'm wondering if everyone else is aligned on your terminology.